trace_context_ext/HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,52 @@
+# Trace Context Extended HTTP Header Format
+
+A correlation context header is used to pass the name-value context properties for the trace. This is a companion header for the `Trace-Context`. The values MUST be passed along to any child requests. Note that uniqueness of the key within the `Trace-Context-Ext` is not guaranteed. Context received from upstream service may be altered before passing it along.
+
+*See [rationale document](HTTP_HEADER_FORMAT_RATIONALE.md) for details of decisions made for this format.*
+
+# Format
+
+## Header name
+
+`Trace-Context-Ext`
+
+Multiple correlation context headers are allowed. Values can be combined in a single header according to the [RFC 7230](https://tools.ietf.org/html/rfc7230#page-24).
+
+## Header value
+
+`name1[=value1[;properties1]],name2[=value2[;properties2]]`
+
+**Limits:**
+Maximum length of a combined header MUST be less than 512 characters. 
```

Comment:
a character has an encoding: for example UTF16, so a lot of work to determine the size in characters. Better to say 512 bytes.